content/install-pop.md

```diff
@@ -22,68 +22,105 @@ tableOfContents: true
 
 The following guide describes how to download the Pop!\_OS.iso image, write it to a flash drive, and install it on the hardware of your choice.
 
-Requirements: At the time of this writing Pop!\_OS only runs on 64-bit x86 architecture, 2 GB of RAM is required, 4 GB of RAM is recommended and 20 GB of storage is recommended.
+## Requirements
 
-## Download the Pop!\_OS.iso image
+### System Specifications
 
-From <https://pop.system76.com/>, choose **DOWNLOAD**, and then choose from the current release (default), or the Long Term Support (LTS) image. The download dialog includes the current _Requirements_, and download _Filesize_, as well as the image checksums used to verify the file once downloaded.
+| Component | Requirement | Recommended |
+|-----------|-------------|-------------|
+| CPU       | 64-bit x86 architecture |
+| RAM       | 2 GB        | 4 GB        |
+| Drive Storage | 20 GB   |             |
 
-## Make Bootable Drive
+### Installation Media
 
-Please see our instructions for making and booting from a live disk of Pop!\_OS [here](/articles/live-disk/).
+The Pop!\_OS.iso can be easily flashed to a USB drive. See our [Live Disk Creation](/articles/live-disk/) article for instructions to create a bootable USB drive in Windows, macOS, and Linux.
+
+### Secure Boot
+
+Systems with Secure Boot enabled must disable this feature before installing Pop!\_OS. Secure boot can be disabled in the BIOS of most computers; however, the process to disable secure boot will vary by laptop and motherboard model.
```

Comment:
>Systems with Secure Boot enabled must disable this feature

This sentence feels a little awkward to me. Maybe it could be streamlined to something like:

>Secure boot must be disabled before installing Pop!\_OS

Reply:
Sounds good. I assumed there was some significant number of machines that did not have a secure boot.